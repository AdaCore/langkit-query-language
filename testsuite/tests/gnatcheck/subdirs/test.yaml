driver: gnatcheck
format: short
subdirs: custom_subdirs
project: prj.gpr
rules:
  - +RGOTO_Statements
  - +RWarnings:a
post_python: ls("custom_subdirs/gnatcheck")
