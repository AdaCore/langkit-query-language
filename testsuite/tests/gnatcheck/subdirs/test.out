gnatcheck: error when calling gprbuild:

   compilation of main.adb failed

gprbuild: *** compilation phase failed
   no exempted violations detected
main.adb:3:05: goto statement
   no rule exemption problems detected
main.adb:6:05: error: "Put_Line" is undefined
main.adb:6:05: error: possible missing "with Ada.Text_IO; use Ada.Text_IO"
   no internal error detected
Content of custom_subdirs/gnatcheck : ['restriction_pragmas.adc', 'main.adb.stderr', 'main.adb.stdout']
